strawberry/field.py

```diff
@@ -39,6 +40,8 @@
 
 _RESOLVER_TYPE = Union[StrawberryResolver, Callable, staticmethod, classmethod]
 
+UNRESOLVED = sentinel.create("UNRESOLVED")
```

Comment:
I think we removed the sentinel package recently, can you rebase? 

maybe we create our own sentinel without add the package back again :)